### DISCUSSION TITLE
Start and Search not working

### DISCUSSION THREAD

**Discussion by RickGre** [valinet/ExplorerPatcher#3701] [Q&A] [answered]
After trying to upgrade Explorer Patcher (EP) to the latest version, Norton 360 prevented a file from installing, thus caused Start and Search not to work. 
  Current v of EP is 22621.3880.66.5

I see that the current release is 22621.3880.66.6 

What do you think about uninstalling EP and reinstalling the current version?

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3228

Please close this.

MS just doesn't like EP and has added it to windows defender, even though it is not a virus and not a hack tool.

All builds of EP are done by github build servers against the source code in the repo.  Only binary is the reimplemented windows 10 taskbar and windows 10 start menu files.  There is no third party build servers.

See the release notes here for the powershell script you can run that excludes all the directories necessary in windows defender: https://github.com/valinet/ExplorerPatcher/releases

The powershell script you must run with elevated privileges.

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

if you get the error: 0x800106ba, it means Windows Defender is not active. You may have another antivirus program active.

**Comment by RickGre** [comment, marked as answer]
Thank you.

Because the message I received focused on Norton I added the following to Norton 360 Exclusions.

 

In term of Windows Defender. I do not believe I received a message referencing that; but if this reoccurs, I will follow your recommendations.

 





 

I’m new to Github. I’ll close this ticket ASAP

**Comment by RickGre** [comment]
Thank you for the help.
